modules/47_regipol/none/bounds.gms

```diff
@@ -42,4 +42,27 @@ vm_capEarlyReti.up('2025',regi,'pc') = 0.65;
 vm_deltaCap.up(t,regi,"gasftrec",rlf)$((t.val gt 2005) and (regi_group("EUR_regi",regi))) = 0;
 vm_deltaCap.up(t,regi,"gasftcrec",rlf)$((t.val gt 2005) and (regi_group("EUR_regi",regi))) = 0;
 
+*** China-specific brownfield planning pipeline 
```

Comment:
actually (writing this after I wrote the comment further down), I would clarify here "This adjustment of coal capacity and vm_capFac is meant to give the PYPSA-coupled REMIND version a better starting point. Before turning this switch on in normal REMIND runs, double-check the electricity sector results that come out for China"